app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (16, 16, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (30, 20, [3, 1])
+    ],
+    ids=[
+        "both zero age",
+        "both below first year",
+        "both exactly first year",
+        "slightly above first year",
+        "both exactly second year",
+        "both slightly above second year",
+        "cat older than dog",
+        "both much older",
+        "cat and dog different ages"
+    ]
+)
+def test_get_human_age_valid(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
```

Comment:
It's a good practice to add a docstring to your functions. It allows other developers to understand the purpose of your function, arguments it expects and what it returns. For example:
python
"""
This function tests the get_human_age function with valid inputs.

:param cat_age: The age of the cat in cat years.
:param dog_age: The age of the dog in dog years.
:param expected: The expected output list of ages in human years.
"""
```

Reply:
Done.